docs/ORGANISATION_CSV_IMPORT.md

```diff
@@ -0,0 +1,317 @@
+# Organisation CSV Import - Admin Documentation
+
+## Overview
+
+The `OrganisationAdmin` class provides CSV-based bulk import functionality for managing organisations and their associated users (for the Cyber Advisor role) within the webcaf application. This feature is accessible through the Django admin interface and allows administrators to efficiently create or update multiple organisations and assign cyber advisors in a single operation.
+
+## Features
+
+- **CSV Template Download**: Generate a standardized CSV template with required headers
+- **Bulk Organisation Import**: Create or update multiple organisations from a CSV file
+- **User Management**: Automatically create or associate cyber advisor users with organisations
+- **Parent Organisation Relationships**: Establish hierarchical relationships between organisations
+- **Organisation Type Support**: Set organisation types during import
+- **Smart Matching**: Find existing organisations by reference or name before creating duplicates
+
+---
+
+## CSV Template Download
+
+### Accessing the Template
+
+Navigate to the Django admin interface:
+1. Go to `/admin/webcaf/organisation/` (Click organisation link on the left navigation pane)
+2. Access the template download via: `/admin/webcaf/organisation/import-org-csv-template/` (Available from the "Import Organisations" screen)
+
+Click the "Import Organisations from csv" button to access the import page:
+![img.png](img.png)
+### Template Structure
+
+The downloaded template contains the following headers:
+
+| Column | Description | Required | Example |
+|--------|-------------|----------|---------|
+| `Organisation` | Name of the organisation | Yes | "Ministry of Example" |
+| `Lead Government Department` | Parent organisation name | No | "Cabinet Office" |
+| `Reference` | Unique organisation reference code | No | "ORG001" |
```

Comment:
Could we add some instructions so a user can know where to find the organisation reference if ones exists and they want to add it